src/ffi/c.rs

```diff
@@ -75,54 +87,65 @@ impl Drop for StreamWrapper {
     }
 }
 
-const ALIGN: usize = std::mem::align_of::<usize>();
+#[cfg(all(feature = "any_zlib", not(feature = "libz-rs-sys")))]
```

Comment:
in the case of zlib-rs, the allocator logic is unused. That is an error for this crate, so it's only conditionally compiled now.